events/c_2_project_and_community.toml

```diff
@@ -64,81 +66,39 @@ This is a track
 # - recommended lunch: 12:00 or 13:00
 # - recommended end:   17:00
 # - use 15m, 20m, 30m, 45m, or 60m session slots
-# - use 15m breaks, one in the morning, one or two in the afternoon
+
 [[timeslots]]
 startTime="09:00"
-speakers=["@speaker_name", "@other_speaker_name"]
+speakers=["@mishmosh"]
 title="Welcome standup discussion"
-description=""
+description="A brief introduction to the track"
 
 [[timeslots]]
-startTime="09:30"
-speakers=["@yourname"]
-title="Track Intro"
-description=""
+startTime="09:15"
+speakers=["@lidel"]
+title="Lightweight RFC Process"
+description="A walkthrough of the new RFC process and its motivations, updated specs repository, and how you can submit an RFC."
 
 [[timeslots]]
-startTime="10:00"
+startTime="09:45"
 speakers=[]
 title="Open Slot"
 description=""
 
 [[timeslots]]
-startTime="10:45"
+startTime="10:15"
 speakers=[]
 title="Open Slot"
 description=""
 
 [[timeslots]]
-startTime="11:30"
-speakers=[]
-title="Break"
-description=""
-
-[[timeslots]]
-startTime="11:45"
-speakers=[]
-title="Open Slot"
-description=""
-
-[[timeslots]]
-startTime="12:30"
-speakers=[]
-title="Open Slot"
-description=""
-
-[[timeslots]]
-startTime="13:00"
-speakers=[]
-title="Lunch Break"
-description=""
-
-[[timeslots]]
-startTime="14:00"
-speakers=[]
-title="Open Slot"
-description=""
-
-[[timeslots]]
-startTime="14:45"
-speakers=[]
-title="Open Slot"
-description=""
-
-[[timeslots]]
-startTime="15:30"
-speakers=[]
-title="Open Slot"
-description=""
-
-[[timeslots]]
-startTime="16:15"
-speakers=[]
-title="Break"
-description=""
+startTime="10:45"
+speakers=["@mishmosh"]
+title="Reorienting the website"
+description="IPFS's public presence predominantly communicates that there are 2 IPFS implementations, js-ipfs and go-ipfs. It also communicates that the project is essentially PL-owned. During this session, we will re-orient some key landing pages toward today's Cambrian explosion of new implementations. This is intended to a fairly surgical co-working session, with a short discussion to kick off and then each participant creating PRs."
 
 [[timeslots]]
-startTime="16:30"
-speakers=[]
-title="Open Slot"
+startTime="11:30"
+speakers=["TBD, maybe @b5 and @mishmosh"]
+title="Fireside chat: Funding and Sustainability"
```

Comment:
In! It'd be great to do some homework for this one to make sure we're leaving folks with useful info. This will be happening after https://fundingthecommons.io/ which might be great to try to build upon.